api/models/service.go

```diff
@@ -70,6 +74,7 @@ func NewService(model *Model, version *Version, modelOpt *ModelOption, endpoint
 		DeploymentMode:    endpoint.DeploymentMode,
 		AutoscalingPolicy: endpoint.AutoscalingPolicy,
 		Protocol:          endpoint.Protocol,
+		CurrentIsvcName:   endpoint.InferenceServiceName,
```

Comment:
I think how we set `CurrentIsvcName` should be different - it is meant to be the currently deployed endpoint's isvc name, and not the name for the new revision, right? If there is no current endpoint, the `CurrentIsvcName` should be empty (`""`). I don't see this happening in the code now.

Reply:
-> t is meant to be the currently deployed endpoint's isvc name, and not the name for the new revision, right?

Correct.

-> If there is no current endpoint, the CurrentIsvcName should be empty (""). I don't see this happening in the code now.

Yes, as suggested, `Service.CurrentIsvcName` will be empty for the first deployment. Once deployment is successful, we set the `Service.Name` to `VersionEndpoint.InferenceServiceName` and save it to DB. So for the next redeployment the `Service.CurrentIsvcName` will use the value from DB.